### ISSUE TITLE
Finish designing and Implement a "Was this helpful?" widget

### ISSUE THREAD

**Issue by kevwalsh** [department-of-veterans-affairs/va.gov-cms#5811]
As a KB manager, i know if my content is helping users. 

<details><summary>Designs so far, screenshot</summary>

Widget ideas

![Linking_from_the_editorial_experience_–_Figma](https://user-images.githubusercontent.com/643678/129255726-1b883be2-1bc4-4325-b394-b5ef0f7d0833.png)

Feedback post widget ideas

![Linking_from_the_editorial_experience_–_Figma](https://user-images.githubusercontent.com/643678/129255783-0963bbeb-c6f5-4791-b617-c96b2404e4c6.png)

</details>

[Link to figma](https://www.figma.com/file/PySxm5aQeWCQvYqZpdkYJj/Linking-from-the-editorial-experience?node-id=0%3A1)

^^^ needs edits. Plz check w/ Cindy & Kauff first before starting!

## ACs
- [x] Find best drupal contrib architecture for this
- [x] evaluate workflow and UI with design team
- [x] We can collect metrics about each node, and available to knowledge base admins
- [x] Additinoal feedback comment box is a nice to have.

**Comment by msbtterswrth** [comment]
A couple of modules we might look at: 

- https://www.drupal.org/project/did_this_help
This one has a nice ui and options for why you voted and allows for a comment. 
![image](https://user-images.githubusercontent.com/8375335/164298657-5fc7456f-0d01-40e4-ba5f-6872cfcf33aa.png)

- https://www.drupal.org/project/rate
This one is widely used and includes several simple up/down ux choices but doesn't allow for comments out of the box (as far as i know).
![image](https://user-images.githubusercontent.com/8375335/164298891-4b9f5a0f-e22a-4fbb-8ac0-73b2d9c9cdd3.png)

- https://www.drupal.org/project/admin_feedback
This module is _very_ similar to the first one but is covered by sec policy and has a slightly different ux. 
![image](https://user-images.githubusercontent.com/8375335/164299142-fac5b04c-9b58-4742-9613-55fe4211dd73.png)

**Comment by msbtterswrth** [comment]
Decisions that came out of meeting with UX folks:

- No cookies
- - Need to accommodate use cases where editors return to an article multiple times. Low chance of folks spamming an article with feedback
- Skip validation for freeform feedback box
- - Not needed yet bc we’re not using it
- Turn off freeform feedback box to start
- - Add helpdesk link for no responses
- - Could add a link to the helpdesk after a no response instead (need to validate with dave)
- Use Admin_feedback module
- - Aligns with our needs, meets security reqs
- Style as secondary icon buttons (stretch - omit if we run out of time)
- Position of last updated - pull out into a separate issue
- - (position at top?, very bottom?)
- Left align for MVP
- - Centering would need more consideration. Let’s wait.
